content/influxdb/cloud/visualize-data/dashboards/create-dashboard.md

```diff
@@ -12,7 +12,6 @@ menu:
 weight: 201
 ---
 
-## Create a new dashboard
```

Comment:
There is one link affected by removing this header. It's in `content/influxdb/v2.0/query-data/flux/sql.md`, line 394.

Reply:
Nice! thank you